[2024-06-28T22:56:16.180+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-28T22:56:16.205+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: NYC_Taxi_pipeline.data_delivery manual__2024-06-28T22:56:11.467865+00:00 [queued]>
[2024-06-28T22:56:16.215+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: NYC_Taxi_pipeline.data_delivery manual__2024-06-28T22:56:11.467865+00:00 [queued]>
[2024-06-28T22:56:16.215+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-28T22:56:16.231+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): data_delivery> on 2024-06-28 22:56:11.467865+00:00
[2024-06-28T22:56:16.240+0000] {standard_task_runner.py:63} INFO - Started process 493 to run task
[2024-06-28T22:56:16.244+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'NYC_Taxi_pipeline', 'data_delivery', 'manual__2024-06-28T22:56:11.467865+00:00', '--job-id', '173', '--raw', '--subdir', 'DAGS_FOLDER/NYC_Taxi_pipeline.py', '--cfg-path', '/tmp/tmpf1zxskb3']
[2024-06-28T22:56:16.247+0000] {standard_task_runner.py:91} INFO - Job 173: Subtask data_delivery
[2024-06-28T22:56:16.295+0000] {task_command.py:426} INFO - Running <TaskInstance: NYC_Taxi_pipeline.data_delivery manual__2024-06-28T22:56:11.467865+00:00 [running]> on host 79a59ce16f65
[2024-06-28T22:56:16.619+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Mahmoud' AIRFLOW_CTX_DAG_ID='NYC_Taxi_pipeline' AIRFLOW_CTX_TASK_ID='data_delivery' AIRFLOW_CTX_EXECUTION_DATE='2024-06-28T22:56:11.467865+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-28T22:56:11.467865+00:00'
[2024-06-28T22:56:16.620+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-28T22:56:16.629+0000] {NYC_Taxi_pipeline.py:54} INFO - data_delivery started
[2024-06-28T22:56:16.629+0000] {Load_Data.py:20} INFO - Loading location Dimension ...
[2024-06-28T22:56:16.637+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-28T22:56:16.640+0000] {Load_Data.py:36} INFO - Existing records columns: ['locationid', 'borough', 'zone', 'service_zone', 'start_date', 'end_date', 'active_flag', 'version']
[2024-06-28T22:56:16.642+0000] {db_utils.py:40} INFO - Error while executing query for table DimLocation: argument 1 must be a string or unicode object: got tuple instead
[2024-06-28T22:56:16.642+0000] {Load_Data.py:61} INFO - update row
[2024-06-28T22:56:16.770+0000] {db_utils.py:40} INFO - Error while executing query for table DimLocation: argument 1 must be a string or unicode object: got tuple instead
[2024-06-28T22:56:16.771+0000] {Load_Data.py:61} INFO - update row
[2024-06-28T22:56:16.772+0000] {db_utils.py:40} INFO - Error while executing query for table DimLocation: argument 1 must be a string or unicode object: got tuple instead
[2024-06-28T22:56:16.772+0000] {Load_Data.py:61} INFO - update row
[2024-06-28T22:56:16.772+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:56:16.773+0000] {Table_creation.py:51} INFO - Loading data into table DimLocation...
[2024-06-28T22:56:16.776+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-28T22:56:16.777+0000] {Table_creation.py:64} INFO - insert new row to dimlocation
[2024-06-28T22:56:16.778+0000] {Table_creation.py:64} INFO - insert new row to dimlocation
[2024-06-28T22:56:16.778+0000] {Table_creation.py:64} INFO - insert new row to dimlocation
[2024-06-28T22:56:16.779+0000] {Table_creation.py:66} INFO - Data loaded successfully into table DimLocation With 3 Row
[2024-06-28T22:56:16.780+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:56:16.780+0000] {Load_Data.py:92} INFO - DimLocation dimension loaded successfully
[2024-06-28T22:56:16.780+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:56:16.780+0000] {Load_Data.py:100} INFO - loading DimRate Dimension ...
[2024-06-28T22:56:16.782+0000] {Table_creation.py:30} INFO - Loading data into table DimRate...
[2024-06-28T22:56:16.785+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-28T22:56:16.791+0000] {Table_creation.py:45} INFO - Data loaded successfully into table DimRate With 6 Row
[2024-06-28T22:56:16.792+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:56:16.792+0000] {Load_Data.py:103} INFO - DimRate_df dimension loaded successfully
[2024-06-28T22:56:16.792+0000] {Load_Data.py:118} INFO - loading payment dimension....
[2024-06-28T22:56:16.793+0000] {Table_creation.py:30} INFO - Loading data into table DimPayment...
[2024-06-28T22:56:16.796+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-28T22:56:16.802+0000] {Table_creation.py:45} INFO - Data loaded successfully into table DimPayment With 6 Row
[2024-06-28T22:56:16.802+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:56:16.802+0000] {Load_Data.py:121} INFO - payment_df dimension loaded successfully
[2024-06-28T22:56:16.803+0000] {Load_Data.py:11} INFO - loading Dimvendor Dimension ...
[2024-06-28T22:56:16.804+0000] {Table_creation.py:30} INFO - Loading data into table DimVendor...
[2024-06-28T22:56:16.807+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-28T22:56:16.812+0000] {Table_creation.py:45} INFO - Data loaded successfully into table DimVendor With 2 Row
[2024-06-28T22:56:16.813+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:56:16.813+0000] {Load_Data.py:14} INFO - Dimvedor_df dimension loaded successfully
[2024-06-28T22:56:16.813+0000] {Load_Data.py:109} INFO - loading trip_type dimension....
[2024-06-28T22:56:16.815+0000] {Table_creation.py:30} INFO - Loading data into table DimTrip_type...
[2024-06-28T22:56:16.818+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-28T22:56:16.823+0000] {Table_creation.py:45} INFO - Data loaded successfully into table DimTrip_type With 2 Row
[2024-06-28T22:56:16.823+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:56:16.823+0000] {Load_Data.py:112} INFO - trip_df dimension loaded successfully
[2024-06-28T22:56:16.824+0000] {Load_Data.py:127} INFO - loading fact table ....
[2024-06-28T22:56:20.952+0000] {Table_creation.py:30} INFO - Loading data into table Fact_Trip...
[2024-06-28T22:56:20.956+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-28T22:59:19.575+0000] {Table_creation.py:45} INFO - Data loaded successfully into table Fact_Trip With 618792 Row
[2024-06-28T22:59:19.576+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:59:24.529+0000] {Load_Data.py:149} INFO - Fact_df  loaded successfully
[2024-06-28T22:59:24.561+0000] {NYC_Taxi_pipeline.py:61} INFO - data_delivery done
[2024-06-28T22:59:24.561+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-28T22:59:24.562+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-28T22:59:24.571+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=NYC_Taxi_pipeline, task_id=data_delivery, run_id=manual__2024-06-28T22:56:11.467865+00:00, execution_date=20240628T225611, start_date=20240628T225616, end_date=20240628T225924
[2024-06-28T22:59:24.615+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-28T22:59:24.630+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-28T22:59:24.631+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
